vms/pom/pages/eventSignUpPage.py

```diff
@@ -3,7 +3,6 @@
 from pom.locators.eventSignUpPageLocators import EventSignUpPageLocators
 from pom.pages.homePage import HomePage
 
-
```

Comment:
Shouldn't there be 2 blank lines between class and imports according to pep8 guidelines?

Reply:
deletion is still here, can you check it once?